core/config/envvar/schema.go

```diff
@@ -36,6 +36,7 @@ type ConfigSchema struct {
 	TelemetryIngressLogging      bool            `env:"TELEMETRY_INGRESS_LOGGING" default:"false"`
 	TelemetryIngressServerPubKey string          `env:"TELEMETRY_INGRESS_SERVER_PUB_KEY"`
 	TelemetryIngressURL          *url.URL        `env:"TELEMETRY_INGRESS_URL"`
+	ShutdownGracePeriod          time.Duration   `env:"SHUTDOWN_GRACE_PERIOD" default:"15s"`
```

Comment:
Its 30s in the test but 15s here?

Reply:
This would be the default for prod/dev, not for tests. And, in according to @jmank88 comment above, I need to reduce this to 5s and see why we could not achieve this timing in CI integration tests.